cogs/misc.py

```diff
@@ -58,10 +58,14 @@ def __init__(self, bot):
             em.add_field(name=u'\U0001F4E5 Msgs received', value=str(self.bot.message_count))
             em.add_field(name=u'\u2757 Mentions', value=str(self.bot.mention_count))
             em.add_field(name=u'\u2694 Servers', value=str(len(self.bot.guilds)))
-            em.add_field(name=u'\ud83d\udcd1 Channels', value=str(channel_count))
+            em.add_field(name=u'\ud83d\udcd1 Channels', value=str(channel_count)+' ('+str(len(self.bot.private_channels))+' private)')
```

Comment:
Is the private channel count really necessary? It might cause additional confusion to users. Mine, for example, is `88`.